### DISCUSSION TITLE
Marker size

### DISCUSSION THREAD

**Discussion by Diewy** [themustafaomar/jsvectormap#97] [Q&A] [answered]
Hi Mustafa Omar,

Quick question, is there a way to set the marker size?

Thanks!
Dieter

**Comment by themustafaomar** [comment]
Hi @Diewy 

Yes you can set `r`  property in the `markerStyle` object.

```js
const map = new jsVectorMap({
  markerStyle: {
    initial: { r: 15 }
  }
})
```

Demo: https://codepen.io/themustafaomar/pen/ZExoBqB

**Comment by Diewy** [comment]
Hi,

Thanks for your fast answer!
I'm using image markers but can't find any attributes that do the same?

Thanks

**Comment by themustafaomar** [comment, marked as answer]
Looks like the image marker config doesn't receive any custom options except for `image` url, of course gonna fix this on the next release.

However, as a workaround we can rely on CSS for now.

https://codepen.io/themustafaomar/pen/ZExoBqB

```css
#jvm-markers-group image {
  width: 35px;
  height: 35px;
}
```

**Comment by Diewy** [comment]
Thank you!

Great project, keep it up. Consider enabling the "Sponsor" button via the GitHub settings. 😉

**Reply by themustafaomar** [reply]
Thank you for the kind words, really appreciate it!
